constantinCurtis/src/main/java/README.md

```diff
@@ -1,10 +0,0 @@
-1. The current repository contains 5 packages. 1 Package per student.
```

Comment:
deleted file, are you sure?

Reply:
Yes. File README.md should be deleted. Andrei Secu confirmed.